_includes/next_meetup.html

```diff
@@ -0,0 +1,65 @@
+{% assign meetup_date = '15 de septiembre' %}
+{% assign meetup_time = '19 hs' %}
+{% assign meetup_place = 'Eagerworks | Juan Paullier 1218' %}
+{% assign meetup_first_speaker_name = 'Marcos Bellucci' %}
+{% assign meetup_first_speaker_talk = 'Ruby on Wasm' %}
+{% assign meetup_second_speaker_name = 'Marcos Bellucci' %}
+{% assign meetup_second_speaker_talk = 'Ruby on Wasm' %}
```

Comment:
@santib Do you know if this is the correct way of assigning variables in an "partial"? I think front matter doesn't work here but I'm not super familiar with Jekyll.

Reply:
I don't have much experience with Jekyll either, but seems that includes files are not intended to define front matter, so I'm not sure if there is a better option than what you did here. @elcuervo might know better

Reply:
Another way would be to have this as a JSON data file: https://jekyllrb.com/docs/datafiles/
Same as we are doing with https://github.com/rubyuy/ruby.uy/blob/main/_data/companies.yml.

Reply:
Thanks for the help, I extracted the data into a datafile.

Reply:
changes look great 😄